Asgardia.Constitution.Proposal.txt

```diff
@@ -82,7 +82,7 @@ Article 5. Territory
 § 1. At the time of its foundation, Asgardia’s territory is the digitised noosphere, a digital state with living citizens on Earth, which exists on a near-Earth orbit, in the form of a single satellite or an orbital satellite constellation.

```

Comment:
Yeah, I had some trouble keeping that consistent. The original doc was composed in UK English whereas I learned US English. 😄